gradio/cli/commands/components/build.py

```diff
@@ -25,6 +30,29 @@ def _build(
         )
         if build_frontend:
             live.update(":art: Building frontend")
+            component_directory = path.resolve()
+
+            node = shutil.which("node")
+            if not node:
+                raise ValueError("node must be installed in order to run dev mode.")
+
+            node_cmds = [
+                node,
+                gradio_node_path,
+                "--component-directory",
+                component_directory,
+                "--root",
+                gradio_template_path,
+                "--mode",
+                "build",
+            ]
+
+            pipe = subprocess.run(node_cmds, capture_output=True, text=True)
+            if pipe.returncode != 0:
+                live.update(":red_square: Build failed!")
+                live.update(pipe.stderr)
+            else:
+                live.update(":white_check_mark: Build succeeded!")
```

Comment:
Should we pass the `sys.executable` path down to the node script and use it to call any python scripts?

Reply:
That's a good point. Yea I think we should be using the full path just cause I suspect it will cause issues on windows if we don't. That's for both `python` and `gradio` commands.